custom-tests.yaml

```
@@ -1513,7 +1513,7 @@ api:
       <%api.RTCDataChannel:channel%>
       var instance = new RTCDataChannelEvent('', {channel: channel});
   RTCDTMFToneChangeEvent:
-    __base: "new RTCDTMFToneChangeEvent('', {tone: ''});"
+    __base: var instance = new RTCDTMFToneChangeEvent('', {tone: ''});
```

Actually let's also add an event type similar to most other events:

```suggestion
    __base: var instance = new RTCDTMFToneChangeEvent('tonechange', {tone: ''});
```